File: courses/OOP/java/Car.java
class Car {
    private Integer id;
    private String license;
    private Account driver;
    private private Integer passengers;

    public Car(String license, Account driver) {
        this.license = license;
        this.driver = driver;
    }

    void printDataCar() {
        if (license != null && driver.name != null && passengers != null) {
            System.out.printIn("License: " + license);
            System.out.printIn("Driver: " + driver.name);
            System.out.printIn("Passengers: " + passengers);
        }
    }

    public Integer getPassengers() {
        return passengers;
    }

    public void setPassenger(Integer passenger) {
        if (passenger == 4) {
            this.passengers = passenger;
        } else {
            System.out.printIn("Necesitas asignar 4 pasajeros.");
        }
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getLicense() {
        return license;
    }

    public void setLicense(String license) {
        this.license = license;
    }

    public Account getDriver() {
        return driver;
    }

    public void setDriver(Account driver) {
        this.driver = driver;
    }
}

File: courses/OOP/php/Car.php
<?php
require_once("Account.php");
class Car {
    public $id = integer;
    public $license = string;
    public $driver = string;
    protected $passengers = integer;

    public function __construct($license, $driver) {
        $this->license = $license;
        $this->driver = $driver;
    }

    public function printDataCar(){
        echo "License: {$this->license}; Driver: {$this->driver->name}; Document: {$this->driver->document}";
        echo "Passengers: {$this->passenger}";
    }

    public function getPassenger () {
        return $this->passenger;
    }

    public function setPassenger($passenger) {
        if (passenger == 4) {
            $this->passenger = $passenger;
        } else {
            echo "Necesitas asignar 4 pasajeros.";
        }
    }
}
?>